jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.editor
+
+import com.intellij.util.ui.ColumnInfo
+import com.intellij.util.ui.ListTableModel
+import software.amazon.awssdk.services.cloudwatchlogs.model.LogStream
+import software.aws.toolkits.resources.message
+import java.time.Instant
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeFormatter.ISO_LOCAL_DATE
+import java.time.format.DateTimeFormatter.ISO_LOCAL_TIME
+import java.time.format.DateTimeFormatterBuilder
+import javax.swing.SortOrder
+import javax.swing.table.TableRowSorter
+
+class CloudWatchLogsStreamsColumn : ColumnInfo<LogStream, String>(message("cloudwatch.logs.log_streams")) {
+    override fun valueOf(item: LogStream?): String? = item?.logStreamName()
+}
+
+class CloudWatchLogsStreamsColumnDate : ColumnInfo<LogStream, String>(message("cloudwatch.logs.last_event_time")) {
+    override fun valueOf(item: LogStream?): String? {
+        item ?: return null
+        return dateFormatter.format(Instant.ofEpochMilli(item.lastEventTimestamp()).atZone(ZoneId.systemDefault()))
+    }
+}
+
+class LogGroupTableSorter(model: ListTableModel<LogStream>) : TableRowSorter<ListTableModel<LogStream>>(model) {
+    init {
+        sortKeys = listOf(SortKey(1, SortOrder.DESCENDING))
+        setSortable(0, false)
+        setSortable(1, false)
+    }
+}
+
+private val dateFormatter: DateTimeFormatter = DateTimeFormatterBuilder()
```

Comment:
Since this is user facing, use the OS's format? [DateTimeFormatter#ofLocalizedDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ofLocalizedDateTime-java.time.format.FormatStyle-) Might make sort a bit harder though